quill-finagle-mysql/src/main/scala/io/getquill/sources/finagle/mysql/FinagleMysqlSource.scala

```diff
@@ -1,31 +1,18 @@
 package io.getquill.sources.finagle.mysql
 
-import java.util.TimeZone
-
-import com.twitter.finagle.exp.mysql.Client
-import com.twitter.finagle.exp.mysql.Parameter
-import com.twitter.finagle.exp.mysql.Result
-import com.twitter.finagle.exp.mysql.Row
-import com.twitter.util.Future
-import com.twitter.util.Local
-import com.typesafe.scalalogging.StrictLogging
+import com.twitter.finagle.exp.mysql.{ Client, Parameter, Result, Row }
+import com.twitter.util.{ Await, Future }
+import com.typesafe.scalalogging.Logger
+import io.getquill.FinagleMysqlSourceConfig
 import io.getquill.naming.NamingStrategy
-import io.getquill.sources.sql.{ SqlBindedStatementBuilder, SqlSource }
+import io.getquill.sources.BindedStatementBuilder
 import io.getquill.sources.sql.idiom.MySQLDialect
-import scala.util.Success
+import io.getquill.sources.sql.{ SqlBindedStatementBuilder, SqlSource }
+import org.slf4j.LoggerFactory
 
-import com.twitter.util.Await
 import scala.util.Try
 
-import com.twitter.finagle.Service
-import com.twitter.finagle.exp.mysql.Request
-import com.twitter.finagle.exp.mysql.PrepareRequest
-import io.getquill.FinagleMysqlSourceConfig
-import io.getquill.sources.BindedStatementBuilder
-import com.typesafe.scalalogging.Logger
-import org.slf4j.LoggerFactory
-
-class FinagleMysqlSource[N <: NamingStrategy](config: FinagleMysqlSourceConfig[N])
+abstract class FinagleMysqlSource[N <: NamingStrategy](config: FinagleMysqlSourceConfig[N])
```

Comment:
It has the information [about dateTimezone](https://github.com/juliano/quill/blob/8ac807f1f860cbebd734087140d8dae5cb607897/quill-finagle-mysql/src/main/scala/io/getquill/sources/finagle/mysql/FinagleMysqlSource.scala#L34)